pkg/controller/multiarchbuildconfig/multiarchbuildconfig_test.go

```diff
@@ -41,6 +42,29 @@ func init() {
 	}
 }
 
+type testLogger struct {
```

Comment:
Because of this solely
```golang
func (log *testLogger) Output() string {
	return log.Logger.Out.(*strings.Builder).String()
}
```

Reply:
Why do you need to check the log output on tests? I don't see the point. Also, the whole approach is wrong since you can use the logrus hooks to test this out.

Reply:
I'll assess the hooks and see how the code will look like when using them. To answer your question: https://github.com/openshift/ci-tools/pull/3910#discussion_r1463297795